File: 1_Year/CursoJava/fliper.java
import java.util.Scanner;

public class fliper {
    public static void main(String[] args) 
    {
        Scanner scanf = new Scanner(System.in);
        int x = scanf.nextInt();
        int y = scanf.nextInt();

        if (x==0)
            System.out.println("C");

        if (x==1 && y==0) 
            System.out.println("B");

        if (x==1 && y==1)
            System.out.println("A");
        scanf.close();
    }    
}

File: 1_Year/AED l/www.beecrowd.com.br/Lista 1/Fliper.c
#include <stdio.h>
 
int main() {
 int p, r;

    scanf("%i",&p);
    scanf("%i",&r);

    if ((p==1) && (r==1)) {
      printf("A\n");
    } else if (p==1 && r==0) {
      printf("B\n");
    } else if (p==0) {
      printf("C\n");
    }
 /*
1 1 = A
1 0 = B
0 0 = C
 */
    return 0;
}